package.json

```diff
@@ -1,131 +1,59 @@
 {
-  "name": "software-engineering-daily-website",
+  "name": "sedaily-front-end_nuxt",
   "version": "1.0.0",
-  "description": "A Vue.js project",
-  "author": "TheHollidayInn <krh121791@gmail.com>",
+  "description": "Software Engineering Daily",
+  "author": "dfcook <dfcook@hotmail.com>",
   "private": true,
   "scripts": {
-    "launch": "npm run build && npm start",
-    "dev": "node build/dev-server.js",
-    "dev:api-test": "cross-env API_ENV=testing npm run dev",
-    "dev:api-prod": "cross-env API_ENV=production npm run dev",
-    "prod": "npm run build && node dist/server.js",
-    "start": "npm run dev:api-prod",
-    "build": "node build/build.js",
-    "build-staging": "cross-env API_ENV=testing npm run build",
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
-    "e2e": "node test/e2e/runner.js",
-    "test": "npm run unit && npm run e2e",
-    "selenium-test": "cross-env BABEL_ENV=test mocha test/selenium --timeout 900000",
-    "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs",
-    "lint:fix": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs --fix",
-    "deploy": "git subtree push --prefix dist heroku master",
-    "staging-deploy2": "git push staging `git subtree split --prefix dist master`:master --force",
-    "deploy2": "git push heroku `git subtree split --prefix dist master`:master --force"
+    "nuxt": "nuxt",
+    "dev": "cross-env API_ENV=development npm run nuxt",
+    "dev:api-prod": "cross-env API_ENV=production npm run nuxt",
+    "dev:api-test": "cross-env API_ENV=staging npm run nuxt",
+    "build": "nuxt build",
+    "build:staging": "cross-env API_ENV=staging npm run build",
+    "start": "nuxt start",
+    "generate": "nuxt generate",
+    "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
+    "lint:fix": "eslint --ext .js,.vue --ignore-path .gitignore . --fix",
+    "test": "ava --serial --verbose",
+    "precommit": "npm run lint"
+  },
```

Comment:
We might want:   "heroku-postbuild": "npm run build"
per the heroku nuxt guide: https://nuxtjs.org/faq/heroku-deployment